record-editor/src/app/core/services/record-api.service.ts

```diff
@@ -71,6 +74,42 @@ export class RecordApiService extends CommonApiService {
       .catch((error) => Observable.throw(new ApiError(error)));
   }
 
+  async closeCuratedRecordTickets(
+    record: object,
+    pidType: string,
+    pidValue: string | number
+  ) {
+    if (!record['curated']) {
+      return;
+    }
+    try {
+      const tickets = await this.apiService.fetchRecordTickets(
+        pidType,
+        pidValue
+      );
+      const filteredTickets = tickets.filter((ticket) => {
+        return (ticket as SnowTicket).u_functional_category !== undefined
+          ? ticket['u_functional_category'] === 'arXiv curation' ||
+              ticket['u_functional_category'] === 'Publisher curation'
+          : ticket['queue'] === 'arXiv curation' ||
+              ticket['queue'] === 'Publisher curation';
+      });
+      for (const ticket of filteredTickets) {
+        try {
+          await this.apiService.resolveTicket(pidValue, ticket['sys_id']);
+        } catch (error) {
+          console.error('Could not close ticket', ticket['sys_id']);
+        }
+      }
+    } catch (error) {
+      if (error.status === 403) {
+        console.error('Logged in user cannot access tickets');
+      } else {
+        console.error('Could not load tickets');
+      }
+    }
```

Comment:
On more thing that I missed before, if we don't throw it will not display an error message [here](https://github.com/inspirehep/inspirehep/pull/3229/files#diff-5f33fad6fbd3534ab05c5eb40f989ae204853d7846d9b056fe579f5aa1d60321R158)

Reply:
Was actually on purpose (because I don't know how likely it is to have errors here and I didn't wanted to interfere with the "normal" saving workflow)

Reply:
indeed, you have a point, we shouldn't fail saving a workflow because snow is down to close the ticket for example.